src/drt/src/db/tech/frTechObject.h

```diff
@@ -257,10 +257,14 @@ class frTechObject
     logger->info(DRT, 167, "List of default vias:");
     for (auto& layer : layers) {
       if (layer->getType() == dbTechLayerType::CUT
-          && layer->getLayerNum() >= 2 /*BOTTOM_ROUTING_LAYER*/) {
+          && layer->getLayerNum() >= 2 /* BOTTOM_ROUTING_LAYER */) {
         logger->report("  Layer {}", layer->getName());
-        logger->report("    default via: {}",
-                       layer->getDefaultViaDef()->getName());
+        if (layer->getDefaultViaDef() != nullptr) {
```

Comment:
This occurs in the layers below M1 (for the LEF I'm working with) when there are vias defined without any techvias. If this occurs higher up yes, this would be an error, which DRT should probably check for if it doesn't already, but segfaulting in this reporting step should also not occur.

Reply:
I always agree with non-crashing but I'm wondering why you have routing layers below M1.

Reply:
it's vias and it's a great question as they don't appear to be used for anything in the LEF (except to make things harder :-) )

Reply:
If you pass -bottom_routing_layer they shouldn't be used.